README.md

```diff
@@ -1,3 +1,147 @@
+
+After [installing](#installation)) the app:
+1. Open your OpsGenie profile to get your credentials and link to your Mattermost instance.
+2. Select the **Integrations** tab in the **Settings** menu. Then, click on the **Add Integration** button.
+3. Inside the **Add Integration** menu, select the **API** option.
+4. Update the integration name and access. Finish the **API** integration set up clicking the **Save Integration** button.
+5. Copy the given API Key.
+6. Return to Mattermost. 
+7. As a Mattermost system admin user, run the ``/genie configure`` command.
+8. In the configuration modal, enter your API Key.
+
+# Admin guide
+
+## Slash commands
+
+- ``/genie configure``: This command will enable all the other commands; it asks the administrator for an API key (which will be used to execute calls to OpsGenie’s API).
+
+# End user guide
+
+## Get started
+
+## Use ``/genie`` commands
+
+- ``/genie help``: This command will show all current commands available for this application.
+- ``/genie alert create``: Allow any user to create a new alert.
+- ``/genie alert note``: Adds a note to an existing alert.
+- ``/genie alert close``: Closes an existing alert.
+- ``/genie alert ack``: Acknowledge an existing alert.
+- ``/genie alert unack``: UnAcknowledge an existing alert.
+- ``/genie alert snooze``: Snooze an existing alert for a period of time.
+- ``/genie alert assign``: Assign an existing to a mattermost team member.
+- ``/genie alert own``: Take ownership of an existing alert (assign alert to yourself).
+- ``/genie alert priority``: Set the priority of an existing alert.
+- ``/genie alert list``: Get a list of the existing alerts.
+- ``/genie team list``: Get a list of the existing teams.
+- ``/genie subscription add``: Creates a new subscription for notifications: choose a team and a channel and get notified of the updates in that team. You can subscribe more than one team per channel.
+- ``/genie subscription list``: Show the list of all subscriptions made in all of your channels.
+- ``/genie subscription remove``: Will allow you to remove a subscription. No more notifications from that team will be received.
+
+# Development environment
+
+## Manual installation
+
+*  Download the latest repository release.
+
+### Run the local development environment
+
+* You need to have installed at least node version 15 and maximum version 18. You can download the latest lts version of node for the required operating system here https://nodejs.org/es/download/
+
+### Install dependencies
+* Move to the project directory or execute ``cd`` command to the project directory and execute ``npm install`` with a terminal to download all dependency libraries.
+
+```
+$ npm install
+```
+
+*  Update the environment configuration file. The ``.env`` file must be modified or added to set the environment variables, it must be in the root of the repository.
+
+```
+file: .env
+
+PROJECT=mattermost-opsgenie-app
+PORT=4002
+HOST=http://localhost:4002
+```
```

Comment:
Can we alternatively use env variables instead of editing `.env`? If so, the README should reflect this